cloudflare/resource_cloudflare_access_group_test.go

```diff
@@ -120,6 +120,35 @@ func TestAccCloudflareAccessGroup_FullConfig(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAccessGroupWithIDP(t *testing.T) {
+	rnd := generateRandomResourceName()
+	groupName := fmt.Sprintf("cloudflare_access_group.%s", rnd)
+	idpName := fmt.Sprintf("cloudflare_access_identity_provider.%s", rnd)
+	team := "Test Team"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckAccount(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudflareAccessGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareAccessGroupWithIDP(accountID, rnd, team),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudflareAccessGroupExists(groupName, &accessGroup),
+					resource.TestCheckResourceAttr(groupName, "account_id", accountID),
+					resource.TestCheckResourceAttr(groupName, "name", rnd),
+					resource.TestCheckResourceAttr(groupName, "include.0.github.account_id", accountID),
+					resource.TestCheckResourceAttr(groupName, "include.0.github.name", idpName),
+					resource.TestCheckResourceAttr(groupName, "include.0.github.team.0", team),
```

Comment:
Once this is in the schema, it will be `teams`

```suggestion
					resource.TestCheckResourceAttr(groupName, "include.0.github.teams.0", team),
```